app/controllers/account_controller.rb

```diff
@@ -213,8 +205,8 @@ def login_get
 
   def login_post
     user_params = params[:user] || {}
-    @login = user_params[:login].to_s
-    @password = user_params[:password].to_s
+    @login = user_params[:login].to_s.strip
+    @password = user_params[:password].to_s.strip
     @remember = user_params[:remember_me] == "1"
     user = User.authenticate(login: @login, password: @password)
     user ||= User.authenticate(login: @login, password: @password.strip)
```

Comment:
@pellaea - Does your new `strip` on the password param make this second `authenticate` request redundant?

Asking because i'm adapting it to the CRUD branch.
```ruby
@login = user_params[:login].to_s.strip
@password = user_params[:password].to_s.strip
@remember = user_params[:remember_me] == "1"
user = User.authenticate(login: @login, password: @password)
user ||= User.authenticate(login: @login, password: @password.strip)
```

Reply:
Oh, would you look at that!  Yes, good catch.